src/script.js

```diff
@@ -70,7 +69,15 @@ const isReminder = function(email, myEmailAddress) {
 		}
 	}
 
-	return allNamesMe;
+	if(options.reminderTreatment === "all") {
+		return allNamesMe;
+	} else if(options.reminderTreatment === "containing-word") {
+		const titleNode = email.querySelector(".bqe");
+		return allNamesMe && titleNode && titleNode.innerText.match(/reminder/i);
+	}
+
+	return false;
+};
```

Comment:
There's an extra '};' here.
`Uncaught SyntaxError: Unexpected token } @ script.js:81`

Reply:
nice catch. Should be fixed.